images/hub/jupyterhub_config.py

```diff
@@ -139,7 +140,8 @@
     org_whitelist = get_config('auth.github.org_whitelist', [])
     if len(org_whitelist) != 0:
         c.GitHubOAuthenticator.github_organization_whitelist = org_whitelist
-        c.GitHubOAuthenticator.scope = ['read:org'] # required for private membership
+        if not auth_scopes:
+            c.OAuthenticator.scope = ['read:org'] # required for private membership
```

Comment:
This feels like the wrong place to embed the logic for default scopes. Maybe this should be left unset since it should work with public members of an org, and an example added to the docs?

Reply:
That makes sense